$OpenBSD: patch-influxdb_tests_dataframe_client_test_py,v 1.1.1.1 2018/09/17 21:15:10 bket Exp $

Skip test_write_points_from_dataframe_with_numeric_precision as it fails with an
AssertionError, which is caused by numpy (>=1.14.0) changing the str and repr of
ndarrays and numpy scalars.

https://github.com/influxdata/influxdb-python/pull/596

Index: influxdb/tests/dataframe_client_test.py
--- influxdb/tests/dataframe_client_test.py.orig
+++ influxdb/tests/dataframe_client_test.py
@@ -388,6 +388,7 @@ class TestDataFrameClient(unittest.TestCase):
 
             self.assertEqual(m.last_request.body, expected)
 
+    @unittest.skip("numpy >=1.14.0 has changed str and repr of ndarrays and numpy scalars")
     def test_write_points_from_dataframe_with_numeric_precision(self):
         """Test write points from df with numeric precision."""
         now = pd.Timestamp('1970-01-01 00:00+00:00')
